docs/README.md

```diff
@@ -30,8 +30,7 @@ class HelloService extends BaseService {
 
 export function HelloPlugin(app: App) {
   // Add `hello.sayHello` method at '/api'
-  app.getSingleton(RPCServiceRegistry)
```

Comment:
This example is deliberately formatted like this to fit on the page and should not be touched